physics/euler_solver_body.hpp

```diff
@@ -91,90 +83,211 @@ EulerSolver<InertialFrame, PrincipalAxesFrame>::EulerSolver(
   CHECK_LE(Square<AngularMomentum>(), Δ₁);
   CHECK_LE(Δ₃, Square<AngularMomentum>());
 
-  // These quantities are NaN in the spherical case, so they be used with care
-  // before we have checked for this case.
-  auto const B₂₃² = I₂ * Δ₃ / I₂₃;
+  // These quantities are NaN in the spherical case, so they must be used with
+  // care before we have checked for this case.
+  auto const B₃₁² = I₃ * Δ₁ / I₃₁;
   auto const B₂₁² = I₂ * Δ₁ / I₂₁;
-  B₁₃_ = Sqrt(I₁ * Δ₃ / I₁₃);
-  B₃₁_ = Sqrt(I₃ * Δ₁ / I₃₁);
+  auto const B₂₃² = I₂ * Δ₃ / I₂₃;
+  auto const B₁₃² = I₁ * Δ₃ / I₁₃;
+  B₁₃_ = Sqrt(B₁₃²);
+  B₃₁_ = Sqrt(B₃₁²);
 
-  auto const G² =  initial_angular_momentum_.Norm²();
+  auto const G² =  m.Norm²();
```

Comment:
We no longer use `G²`.

Compute `G_` in the initializer as `G_(initial_angular_momentum.Norm())`.